crates/krator/src/runtime.rs

```diff
@@ -55,13 +54,37 @@ impl<R: Resource> From<&Event<R>> for PrettyEvent {
     }
 }
 
+#[derive(Debug)]
+enum ObjectEvent<R> {
+    Applied(R),
+    Deleted {
+        name: String,
+        namespace: Option<String>,
+    },
+}
+
+impl<R: Resource> From<&ObjectEvent<R>> for PrettyEvent {
+    fn from(event: &ObjectEvent<R>) -> Self {
+        match event {
+            ObjectEvent::Applied(object) => PrettyEvent::Applied {
+                name: object.name(),
+                namespace: object.namespace(),
+            },
+            ObjectEvent::Deleted { name, namespace } => PrettyEvent::Deleted {
+                name: name.to_string(),
+                namespace: namespace.clone(),
+            },
+        }
+    }
+}
+
```

Comment:
I know it wasn't introduced here, but what is `PrettyEvent` actually used for? Is this just for tracing purposes?

Reply:
Yes, just makes it easier to debug events without dumping the entire object spec.